tests/integration/component.go

```diff
@@ -853,5 +854,28 @@ func componentTests(args ...string) {
 			oc.WaitAndCheckForExistence("is", project, 1)
 			oc.WaitAndCheckForExistence("service", project, 1)
 		})
+
+		It("should delete the component and the owned resources with wait flag", func() {
+			helper.CopyExample(filepath.Join("source", "nodejs"), context)
+			helper.CmdShouldPass("odo", append(args, "create", "nodejs", cmpName, "--app", appName, "--project", project, "--context", context)...)
+			helper.CmdShouldPass("odo", "url", "create", "example-1", "--context", context)
+
+			helper.CmdShouldPass("odo", "storage", "create", "storage-1", "--size", "1Gi", "--path", "/data1", "--context", context)
+			helper.ValidateLocalCmpExist(context, "Type,nodejs", "Name,"+cmpName, "Application,"+appName, "URL,0,Name,example-1")
+			helper.CmdShouldPass("odo", append(args, "push", "--context", context)...)
+
+			helper.CmdShouldPass("odo", "url", "create", "example-2", "--context", context)
+			helper.CmdShouldPass("odo", "storage", "create", "storage-2", "--size", "1Gi", "--path", "/data2", "--context", context)
+			helper.CmdShouldPass("odo", append(args, "push", "--context", context)...)
+
+			// delete with --wait flag
+			helper.CmdShouldPass("odo", append(args, "delete", "-f", "-w", "--context", context)...)
+
+			oc.VerifyResourceDeleted("routes", "example", project)
+			oc.VerifyResourceDeleted("service", cmpName, project)
+			// verify s2i pvc is delete
+			oc.VerifyResourceDeleted("pvc", "s2idata", project)
```

Comment:
there are three PVCs, we need to check if all of them are deleted

```
Running oc with args [oc get pvc --namespace twmwqqprru]
[oc] NAME                  STATUS    VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS    AGE
[oc] mvfzn-hgfxb-s2idata   Bound     pvc-jdslkaj-9f92-ljajlds          1Gi        RWO            gp2-encrypted   1m
[oc] storage-1-hgfxb-pvc   Bound     pvc-dasjldj-9f92-dhfhd           1Gi        RWO            gp2-encrypted   1m
[oc] storage-2-hgfxb-pvc   Bound     pvc-jsalkjd-9f92-lksala           1Gi        RWO            gp2-encrypted   44s
```

Reply:
Thanks for reminding.